docs/Meadow/Getting_Started/Deploying_Meadow/index.md

```diff
@@ -11,7 +11,7 @@ When you receive your Meadow board, it will need to have the latest Meadow.OS up
 2. [macOS](#macos)
 3. [Linux (Debian, Ubuntu)](#linux-debian-ubuntu)
 
-## Windows
+## **Windows**
```

Comment:
Do we need both the bold and the title markup?

Reply:
Forgot to revert that, done!